src/routes/docs/tooling/keyboard/+page.markdoc

```diff
@@ -0,0 +1,39 @@
+---
+layout: article
+title: Keyboard
+description: How to use the keyboard with the optimized Appwrite Console for both efficiency and accessibility, and introducing The Appwriter.
+---
+
+The Appwrite SDK console is optimized with the keyboard using shortcut keys to enable accessibility.  Using shortcut keys access websites without the mouse. 
+
```

Comment:
Might be a good idea to title this `Windows vs Linux vs macOS` explain the differences and then share the table. Or we can even drop the table as I don't see much value in the `Enter` key description, our audience should be familiar with basics